robotiq_85_description/urdf/tool_changer.urdf.xacro

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
-  <xacro:macro name="tool_changer" params="prefix connected_to">
+  <xacro:macro name="tool_changer" params="prefix connected_to rotation">
```

Comment:
If you use the [xacro syntax for parameter defaults and forwarding](https://github.com/ros/xacro/pull/105), do you think you could make it so this modification doesn't break existing robot descriptions that use this macro?

Here's an [example macro definition](https://github.com/ros-controls/ros2_control_demos/blob/ead1af151d92b104774e9061684cf23b52f67507/ros2_control_demo_description/rrbot_description/ros2_control/rrbot_system_position_only.ros2_control.xacro#L4) and an [example usage](https://github.com/ros-controls/ros2_control_demos/blob/ead1af151d92b104774e9061684cf23b52f67507/ros2_control_demo_description/rrbot_description/urdf/rrbot_system_position_only.urdf.xacro#L40-L44).

Reply:
Yes! That worked great. Tested with the picknik UR5 URDF. Did with and without the new arg, and no complaints while loading the tool changer Xacro (and could see the default rotation when it was excluded)

Reply:
Awesome!